challenge_6/python/sarcodian/src/challenge_6_tests.py

```diff
@@ -0,0 +1,28 @@
+# we start by importing the unittest module
+import unittest
+
+# Next lets import our function that we intend to do testing on
+#
+# **We could also import all functions by using * or just import the module
+# itself but for now lets just import the function
+from challenge_6 import ranges, print_range
+
+# lets define our suite of tests as a class and lets inherit from unittest.TestCase
+class TestBinaryMethods(unittest.TestCase):
+
+    # All tests must begin with the test_* naming scheme otherwise unittest won't find it
+    def test_positiveIntegers(self):
+        """ Tests from the READ.me """
+        # Testing input with only 1 return value
+        self.assertEqual(ranges([1, 2]), ["1->2"])
+
+# Testing input with no return value
+        self.assertEqual(ranges([1]), [])
```

Comment:
Style-wise, it is better to contain each test within their own method. You should name the method something representative of the type of test you are running, such as `test_single_element`.